### PR TITLE
Api v2

### PR DESCRIPTION
**Pull request by wkdewey**

### CODE REVIEW

File: .byebug_history
Comment (nichgray) [inline .byebug_history:1]: Should this file be deleted?

File: collections/cody_archive/output/development/html/albert.edward.html
Comment (nichgray) [inline collections/cody_archive/output/development/html/albert.edward.html:1]: This and subsequent files moved to the nested collection folder should be reviewed. Unclear they should be part of this commit.

File: .byebug_history
Comment (wkdewey) [reply .byebug_history:1]: yes

File: collections/cody_archive/output/development/html/albert.edward.html
Comment (wkdewey) [reply collections/cody_archive/output/development/html/albert.edward.html:1]: they should be deleted

**Review by nichgray — APPROVED** [review]
These changes look good to me. (Some of the changes, merged from another branch, have already been reviewed.)

**Pull request merged**